.github/workflows/lighthouse-report.yaml

```diff
@@ -0,0 +1,64 @@
+name: Lighthouse CI
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Install and Build
+        run: |
+          yarn install
+          yarn build
+      - name: Wait for the Netlify Preview
+        uses: jakepartusch/wait-for-netlify-action@v1
+        id: netlify
+        with:
+          site_name: 'keptn-experimental-docs-site'
+      - name: Audit URLs using Lighthouse
+        id: lighthouse_audit
+        uses: treosh/lighthouse-ci-action@v3
+        with:
+          urls: |
+            https://deploy-preview-$PR_NUMBER--keptn-experimental-docs-site.netlify.app/
+            https://deploy-preview-$PR_NUMBER--keptn-experimental-docs-site.netlify.app/introduction
```

Comment:
I still have the same question. Why're we sending just 2 pages instead of all the pages what we generate?

Reply:
@imsingh Yeah we can add multiple URLs here but it will be a hard task to get URLs of all the docs and links on the website and also generate a Lighthouse report for all URLs since there will be a lot of reports. So need a script or any other means to dynamically generate a list of URLs.